projects/storefrontstyles/scss/components/myaccount/order-history/_replenishment-order-history.scss

```diff
@@ -0,0 +1,221 @@
+            padding-top: 1.25rem;
+          }
+
+          &:last-child {
+            padding-bottom: 1.25rem;
+            padding-top: 7px;
+          }
+        }
+      }
+
+      .#{$block}-code {
+        .#{$block}-value {
+          text-decoration: underline;
+        }
+      }
+
+      .#{$block}-code {
+        .#{$block}-value {
+          text-decoration: underline;
+        }
+      }
+
+      .#{$block}-frequency {
+        width: 25%;
+
+        @include media-breakpoint-down(sm) {
+          width: unset;
+        }
+      }
+
+      .#{$block}-total {
+        text-align: end;
+
+        @include media-breakpoint-down(sm) {
+          text-align: start;
+        }
+      }
+
+      .#{$block}-cancel {
+        width: 15%;
+
+        .cx-order-cancel {
+          text-decoration: underline;
+          text-transform: uppercase;
+          margin: 1.25rem 0;
+        }
+
+        @include media-breakpoint-down(sm) {
+          width: unset;
+        }
+      }
+
+      .#{$block}-ellipses {
+        display: inline-block;
+        max-width: 95%;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+      }
+    }
+
+    &-header {
+      padding: 40px 0 0 0;
+      color: var(--cx-color-text);
+
+      @include media-breakpoint-down(sm) {
+        padding: 40px 20px 0 20px;
+      }
+    }
+
+    &-thead-mobile {
+      @include media-breakpoint-down(sm) {
+        display: none;
+      }
+    }
+
+    &-label {
+      text-transform: uppercase;
+      color: var(--cx-color-secondary);
+
+      @include media-breakpoint-down(sm) {
+        @include type('8');
+        min-width: 150px;
+        margin-block-end: 10px;
+      }
+    }
+
+    &-value {
+      color: var(--cx-color-text);
+
+      @include media-breakpoint-down(sm) {
+        @include type('5');
+        font-weight: $font-weight-normal;
+      }
+    }
+
+    &-form-group {
+      padding: 0;
+      margin-bottom: 0;
+
+      @include media-breakpoint-down(sm) {
+        padding: 1.25rem;
+      }
+    }
+
+    &-sort {
+      &.top {
+        display: flex;
+        justify-content: space-between;
+        padding: 1rem 0;
+        margin: 0;
+
+        @include media-breakpoint-down(sm) {
+          flex-direction: column;
+          padding-top: 0;
+        }
+      }
+
+      &.bottom {
+        display: flex;
+        justify-content: flex-end;
+        padding: 2rem 0 1rem 0;
+        margin: 0;
+
+        @include media-breakpoint-down(sm) {
+          flex-direction: column;
+          padding-top: 1.25rem;
+        }
+      }
+    }
+
+    &-pagination {
+      @include media-breakpoint-down(sm) {
+        margin: 0 auto;
+      }
+    }
+
+    &-no-order {
+      @include type('5');
+      font-weight: $font-weight-normal;
+      min-height: 415px;
+
+      @include media-breakpoint-down(sm) {
+        min-height: 474px;
+        padding-inline-start: 1.25rem;
+        padding-inline-end: 1.25rem;
+      }
+    }
```

Comment:
need to add margin when on button when no replenishments

just add margin: 1.25rem 0; on the btn

![Screen Shot 2020-09-29 at 2 15 24 PM](https://user-images.githubusercontent.com/29099466/94599554-94b4f280-025e-11eb-8ce0-dd2c77350047.png)
![Screen Shot 2020-09-29 at 2 15 36 PM](https://user-images.githubusercontent.com/29099466/94599556-95e61f80-025e-11eb-9a76-5e9d4ee446ea.png)
![Screen Shot 2020-09-29 at 2 15 44 PM](https://user-images.githubusercontent.com/29099466/94599557-95e61f80-025e-11eb-92ad-401c53344c52.png)